k8s/showcase/vineyard-mars-pytorch/Makefile

```diff
@@ -0,0 +1,44 @@
+# setup the registry
+DEFAULT_REGISTRY := ghcr.io/v6d-io/v6d
```

Comment:
Use `vineyardcloudnative` as the default registery in show cases.

`ghcr.io/v6d-io/v6d` is only used for testing in CI.

Reply:
And note that never pushing CI-only images to dockerhub.

You can image that `vineyardcloudnative` is for public/end-user and `ghcr.io` is for internal/developer.

Reply:
Make sense